src/System.Reflection.MetadataLoadContext/src/FromCoreRt/CoreRtBridge.cs

```diff
@@ -0,0 +1,79 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Reflection.TypeLoading;
+
+//
+// This file shims just enough stuff so that source code copied from Reflection.Core builds with as few source changes
```

Comment:
I think we can just do those (pretty simple) source changes to make this unnecessary. This is unnecessary abstraction overhead.

Reply:
If we remove "FromCoreRt" directory separation then yes we should just make the sources to avoid this.

Reply:
Update: I removed the FromCoreRt separation but for now kept the "bridge" class.